tests/steps/test_models.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from jsonschema import ValidationError
+from pyaml_env import parse_config
+from ruamel.yaml import YAML  # type: ignore
+
+from src.mpyl.steps.models import RunProperties, VersioningProperties
+from tests import root_test_path
+
+yaml = YAML()
+yaml.preserve_quotes = True
+
+
+class TestModels:
+    resource_path = root_test_path / "test_resources"
+
+    def test_should_return_error_if_user_not_defined(self):
+        run_properties_values = parse_config(self.resource_path / "run_properties.yml")
+        config_values = parse_config(self.resource_path / "config.yml")
+
+        del run_properties_values['build']['run']['user']
```

Comment:
I think we should leave these fields out of the yaml and test that.. Not remove the item from the dictionary